### DISCUSSION TITLE
Internal SW versus EMS-ESP

### DISCUSSION THREAD

**Discussion by splitbrain** [emsesp/EMS-ESP32#569] [General]
I'm trying to set up some more sophisticated automation (via Home Assistant) than just monitoring the current values. Eg. I want to turn on and off water heating based on a schedule. However it seems this is somewhat interfering with what my boiler's own software is trying to do.

For example: when I turn `dhw activated` to `off`, a few seconds later it will be back to to `on`. So my question is how do I configure my boiler's thermostat to not interfere with my settings anymore.

I'm running a Buderus Topline/GB162 with a RC35 Thermostat. I put the dials for heating and warm water to `auto`. Are there any recommended settings? I feel like this step is missing in the official documentation.

**Comment by proddy** [comment]
https://emsesp.github.io/docs/#/Troubleshooting?id=changing-a-value-works-at-first-but-is-then-reset-to-it39s-original-value
